config/database.yml

```diff
@@ -27,6 +27,7 @@ development:
 test:
   <<: *default
   database: vss_test
+  username: postgres
```

Comment:
Isn't `postgres` the default username?

Reply:
for some reason it tried to use the concourse username... I'll try again

Reply:
yeah, it's trying to login as `root`

Reply:
coolio, just wondering.